wee_slack.py

```diff
@@ -4101,6 +4130,56 @@ def command_thread(data, current_buffer, args):
 command_thread.completion = '%(threads)'
 
 
+def subscribe_helper(current_buffer, args, usage, api):
+    channel = EVENTROUTER.weechat_controller.buffers[current_buffer]
+    team = channel.team
+
+    if args:
+        msg = get_msg_from_id(channel, args)
+    elif isinstance(channel, SlackThreadChannel):
+        msg = channel.parent_message
+    else:
+        w.prnt('', usage)
+        return w.WEECHAT_RC_ERROR
+
+    s = SlackRequest(team, api,
+            {"channel": channel.identifier, "thread_ts": msg.ts}, channel=channel)
+    EVENTROUTER.receive(s)
+    return w.WEECHAT_RC_OK_EAT
+
+
+@slack_buffer_required
+@utf8_decode
+def command_subscribe(data, current_buffer, args):
+    """
+    /slack subscribe <thread>
+    Subscribe to a thread, so that you are alerted to new messages.  When in a
+    thread buffer, you can omit the thread id.
+    """
+    return subscribe_helper(current_buffer, args, 'Usage: /slack subscribe <thread>', "subscriptions.thread.add")
+
+command_subscribe.completion = '%(threads)'
+
+
+@slack_buffer_required
+@utf8_decode
+def command_unsubscribe(data, current_buffer, args):
+    """
+    /slack unsubscribe <thread>
+    Unsubscribe from a thread that has been previously subscribed to, so that
+    you are not alerted to new messages.  When in a thread buffer, you can omit
+    the thread id.
+    """
+    return subscribe_helper(current_buffer, args, 'Usage: /slack unsubscribe <thread>', "subscriptions.thread.remove")
+
+command_unsubscribe.completion = '%(threads)'
+
+
+def handle_subscriptionsthreadmark(json, eventrouter, team, channel, metadata):
+    dbg(repr(json))
+    if not json["ok"]: w.prnt('', 'Failed to set subscription status.  Check debug log for details.')
```

Comment:
Just print the error directly, instead of sending users to the debug log, like it's done in `handle_reactionsadd`. And I haven't seen any error messages which contains anything other than `ok` and `error`, so you can remove `dbg(repr(json))`.